lib/jazzy/source_declaration.rb

```diff
@@ -41,6 +41,13 @@ def fully_qualified_name
       namespace_path.map(&:name).join('.')
     end
 
+    # If this declaration is an objc category, returns an array with the name
+    # of the extended objc class and the category name itself, i.e.
+    # ["NSString", "MyMethods"], nil otherwise.
+    def objc_category_name
+      name.split(/[\(\)]/) if type.objc_category?
```

Comment:
The name `objc_category_name` implies that this returns a string, not an array — and that seems like the right behavior.